### PR TITLE
feat(all): Add new GraphQL scalar type `Int8`

### PR DESCRIPTION
**Pull request by dotansimha**
## TODO
- [x] Add the new scalar to GraphQL 
- [x] Add the new scalar and data handling to `store` 
- [x] Add unit tests to `store`
- [x] Explore if changes are needed in `graph-ts` (https://github.com/graphprotocol/graph-tooling/pull/1296)
- [x] Figure out CI issues with unit tests
- [x] Fix for https://github.com/graphprotocol/graph-node/pull/4303 and rebase 
- [x] Integration tests
- [x] Merge https://github.com/graphprotocol/graph-tooling/pull/1296
- [x] Release CLI+TS 
- [x] Update integration test to use the latest CLI+TS

### CODE REVIEW

File: graphql/src/schema/meta.graphql [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # GraphQL core functionality
 scalar Boolean
 scalar ID
+""" 4 bytes	int """
```
Comment (lutter) [inline graphql/src/schema/meta.graphql:4]: Probably should say `signed` in the comment, too, like `Int8`

File: graphql/src/values/coercion.rs [hunk 2]
```diff
@@ -42,6 +42,10 @@ impl MaybeCoercible<ScalarType> for q::Value {
                     Err(q::Value::Int(num))
                 }
             }
+            ("Int8", q::Value::Int(num)) => {
+                let n = num.as_i64().ok_or_else(|| q::Value::Int(num.clone()))?;
+                Ok(r::Value::Int(n))
+            }
```
Comment (lutter) [inline graphql/src/values/coercion.rs:48]: I didn't realize that the various values were already using `i64` internally .. wild. We should probably have a ticket to narrow `r::Value::Int` to `i32` at some point.

**Review by lutter — APPROVED** [review]
Nice! Looks great!

File: graphql/src/values/coercion.rs (hunk 2 above)
Comment (dotansimha) [reply graphql/src/values/coercion.rs:48]: Yeah, I wasn't sure if that's the right time to touch this or now 😆 I created a ticket here: https://github.com/graphprotocol/graph-node/issues/4552

**Review by lutter — APPROVED** [review]
This looks great! 

Just change the `unwrap` before committing. Also, for the commit message, something like
```
all: Add new scalar type Int8
```
would be more in line with what we've been doing

But with these two touchups, this is good to merge.

File: store/postgres/src/relational_queries.rs [hunk 3]
```diff
@@ -562,6 +579,7 @@ impl<'a> QueryFragment<Pg> for QueryValue<'a> {
         match self.0 {
             Value::String(s) => match &column_type {
                 ColumnType::String => out.push_bind_param::<Text, _>(s),
+                ColumnType::Int8 => out.push_bind_param::<BigInt, _>(&s.parse::<i64>().unwrap()),
```
[the remark this answers is not in the document]
Comment (dotansimha) [reply store/postgres/src/relational_queries.rs:582]: Yeah, that makes perfect sense. Fixing :)

**Comment by dotansimha** [comment]
Thanks @lutter . Fixed the `unwrap` issue and the other things you mentioned. I'll merge when CI clears 🚀

**Pull request merged**